src/Tasks.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Created by Katarzyna_Bak on 28.03.2017.
+ */
+
+import React, { Component } from 'react';
+import './Tasks.css';
+import './fontello.css';
+import TaskItem from './TaskItem';
+import TaskEdit from './TaskEdit';
+
+
+
+function TaskView(props) {
+
+    const taskArray = [1, 2, 3];
+    const taskItems = taskArray.map((taskNum) =>
+        <TaskItem key={taskNum.toString()} num={taskNum}/>
+    );
+
+    if(props.mode === 0) {
+        return (
+            <section className="tasks">
```

Comment:
It can be extracted to separate component (together with generating `taskItems`), so `TaskView` gets simpler.